### ISSUE TITLE
sinisterdesign.net: breakage

### ISSUE THREAD

**Issue by craigstern** [uBlockOrigin/uAssets#19632]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

HI! I am a game developer with the website sinisterdesign.net. Along the side of the web page, I have little banner images that direct to pages where people can purchase my games. One of my games--Telepath RPG: Servants of God--is sold via the long-established affiliate sales platform BMT Micro (which predates Steam by many years and is entirely trustworthy). 

Yesterday, I noticed that the sidebar image for TRPG:SoG was not appearing and that it could not be clicked in my browser. I contacted my webmaster, who emailed me today to inform me, "Your vendor link to `bmtmicro.com` is being caught by a UBlock Origin blacklist."

Put simply: this is not a malicious link, and the fact that uBlock Origin has blacklisted it will have a direct negative impact on my business. I ask that you please whitelist this link so that it will no longer disrupt the operation of my website.

### URL(s) where the issue occurs.

`sinisterdesign.net`
=>
`https://secure.bmtmicro.com/servlets/Orders.ShoppingCart?CID=10&PRODUCTID=53780002`

### Screenshot(s)

![Sidebar Error](https://github.com/uBlockOrigin/uAssets/assets/24302200/ad33a738-3535-49f9-bdad-d2b3485ec760)

### uBO version

current

### Browser name and version

Firefox version 117.0 (64-bit)

### Settings

None

### Notes

See above.

**Comment by FanOfRootBeer** [comment]
@ryanbr image is hidden

**Comment by okiehsch** [comment]
For the hidden image add
`sinisterdesign.net#@#[href^="https://secure.bmtmicro.com/servlets/"]`
